src/utils/messageValidation.ts

```diff
@@ -1,31 +1,33 @@
-import { logGroupCollapsed, logGroupEnd, logInfo } from '../lib/Logger';
 import {
-  IAddButtonInitialMessageData,
-  IContentComponentClickedMessageData,
-  IContentItemClickedMessageData,
-  IElementClickedMessageData,
-  InsertPosition,
+  type IAddButtonInitialMessageData,
+  type IContentComponentClickedMessageData,
+  type IContentItemClickedMessageData,
+  type IElementClickedMessageData,
+  type InsertPosition,
   InsertPositionPlacement,
-} from '../lib/IFrameCommunicatorTypes';
-import { KSLConfiguration } from './configuration';
-import { ParseResult, ParserTokenKey } from './dataAttributes/parser';
-import { NonEmptyArray } from './typeUtils';
+} from "../lib/IFrameCommunicatorTypes";
+import { logGroupCollapsed, logGroupEnd, logInfo } from "../lib/Logger";
+import type { KSLConfiguration } from "./configuration";
+import type { ParseResult, ParserTokenKey } from "./dataAttributes/parser";
+import type { NonEmptyArray } from "./typeUtils";
 
 type ValidationResult<T> =
   | { success: true; data: T }
   | { success: false; missing: Readonly<NonEmptyArray<ParserTokenKey>> };
 
 const validateConfigurationDataAttributes = (
   data: ParseResult,
-  configurationDataAttributes: KSLConfiguration['defaultDataAttributes']
+  configurationDataAttributes: KSLConfiguration["defaultDataAttributes"],
 ): ValidationResult<{ languageCodename: string; projectId: string }> => {
-  const languageCodename = data.parsed.languageCodename ?? configurationDataAttributes.languageCodename;
+  const languageCodename =
+    data.parsed.languageCodename ?? configurationDataAttributes.languageCodename;
   const projectId = data.parsed.environmentId ?? configurationDataAttributes.environmentId;
 
+  // biome-ignore lint/complexity/useSimplifiedLogicExpression: TODO: This is easier than !(a && b)
```

Comment:
If you think the `||` is easier here just remove the TODO. Otherwise change it and remove the ignore. 😄

Reply:
To be clear, I agree that the `||` version is simpler and I am in favor of removing this rule from the biome config.